css/styles.css

```diff
@@ -143,3 +142,34 @@ cursor: pointer;
     color: #1E90FF;
   }
 }
+/* about */
+.about{
+  height:100%;
+  margin-left: 20%;
```

Comment:
Considering that your side menu does not have 20% as its width. You can not set the margin-left of this container to 20%, consider connecting your CSS properly.